Frameworks/UIKit/UITextField.mm

```diff
@@ -1248,21 +1267,28 @@ - (void)_initPasswordBox:(WXFrameworkElement*)xamlElement {
         __strong UITextField* strongSelf = weakSelf;
         if (strongSelf) {
             strongSelf.text = strongSelf->_passwordBox.password;
-            dispatch_async(dispatch_get_main_queue(),
-                           ^{
-                               [strongSelf sendActionsForControlEvents:UIControlEventEditingChanged];
-                           });
+            dispatch_async(dispatch_get_main_queue(), ^{
+                [strongSelf sendActionsForControlEvents:UIControlEventEditingChanged];
+            });
         }
     }];
 
     // set up focus and keydown handlers
     [self _setupControlGotFocusHandler:_passwordBox];
     [self _setupControlLostFocusHandler:_passwordBox];
     [self _setupControlKeyDownHandler:_passwordBox];
+
+    // Remove the old subview (if it exists)
+    [_subView removeFromSuperview];
+
+    // Create a new view for the xaml element and add it as a subview of ourselves
+    _subView = [[UIView alloc] initWithFrame:self.bounds xamlElement:_passwordBox];
+    [_subView setAutoresizingMask:UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight];
+    [self addSubview:_subView];
 }
 
 // Main entrance to initialize TextField
-- (void)_initTextField:(WXFrameworkElement*)xamlElement {
+- (void)_initUITextField:(WXFrameworkElement*)xamlElement {
     self->_secureModeLock = [NSRecursiveLock new];
 
     // creating dummy button and hidden view so that it can be used to steal/kill the focus for this UITextField
```

Comment:
>so that it can be used to steal/kill the focus for this UITextField [](start = 45, length = 67)

unrelated to this change - this comment seems pretty "light" on details... and its not intuitive why we need to create an invisible button here... will follow up offline